core/src/main/resources/lib/hudson/rssBar.properties

```diff
@@ -22,6 +22,6 @@
 # THE SOFTWARE.
 
 Legend=Icon legend
-forAll=for all
-forFailures=for failures
-justLatestBuilds=for just latest builds
```

Comment:
given these are all being changed all the other translations need removing.

(tbh they all already look broken as the keys are wrong but they should be cleaned up)

Reply:
Done